lib/model/query/audits.js

```diff
@@ -38,9 +38,9 @@ const actionCondition = (action) => {
     // The backup action was logged by a backup script that has been removed.
     // Even though the script has been removed, the audit log entries it logged
     // have not, so we should continue to exclude those.
-    return sql`action not in ('entity.create', 'entity.bulk.create', 'entity.error', 'entity.update.version', 'entity.update.resolve', 'entity.delete', 'entity.restore', 'entity.purge', 'submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update', 'submission.backlog.hold', 'submission.backlog.reprocess', 'submission.backlog.force', 'submission.delete', 'submission.restore', 'backup', 'analytics')`;
+    return sql`action not in ('entity.create', 'entity.bulk.create', 'entity.error', 'entity.update.version', 'entity.update.resolve', 'entity.delete', 'entity.restore', 'entity.purge', 'submission.create', 'submission.update', 'submission.update.version', 'submission.attachment.update', 'submission.backlog.hold', 'submission.backlog.reprocess', 'submission.backlog.force', 'submission.delete', 'submission.restore', 'user.preference.update', 'user.preference.delete', 'backup', 'analytics')`;
   else if (action === 'user')
-    return sql`action in ('user.create', 'user.update', 'user.delete', 'user.assignment.create', 'user.assignment.delete', 'user.session.create')`;
+    return sql`action in ('user.create', 'user.update', 'user.delete', 'user.assignment.create', 'user.assignment.delete', 'user.session.create', 'user.preference.update', 'user.preference.delete')`;
```

Comment:
I think we should probably exclude `user.preference.update` and `user.preference.delete` from this `action=user` filter as well. Users can filter for that from the Frontend if they select "Web User Actions".

Alternatively, if it seems too weird to exclude them, I think we'd need to teach Frontend how to display them for if the user selects "Web User Actions".